src/pages/student/Bookmarks.js

```diff
@@ -235,6 +235,13 @@ function Bookmarks({
               alt="flyer bears image"
             />
           </div>
+          <div className="dashboard-events-photo-blob">
+            <img
+              className="dashboard-blobblue-img"
+              src={require('../assets/blobblue.png').default}
+              alt="blob blue"
+            />
+          </div>
```

Comment:
Again see comments in Dashboard.js and Dashboard.scss — the same applies to the corresponding Bookmarks files. Also please rename **dashboard-blobblue-img -> bookmarks-blobblue-img** :D